lib/db/lowlevel.go

```diff
@@ -331,6 +350,14 @@ func (db *Lowlevel) dropFolder(folder []byte) error {
 		return err
 	}
 
+	k5, err := db.keyer.GenerateBlockListMapKey(nil, folder, nil, nil)
```

Comment:
Is there a reason to not reuse byte slices in this function (except that it's already like that now)? Not asking to have it changed, just curious because I don't see it and if there is a reason, I'd like to know in case I ever touch this :)

Reply:
it's not something that is done often, so I guess it does not matter, but I've changed it never the less.